src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userservice = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User saveduser = userservice.findByEmail(email);
```

Comment:
we are not doing any saving logic here, so I should name it just `user`
```suggestion
        User user = userservice.findByEmail(email);
```